src/otx/v2/adapters/torch/modules/utils/mask_to_bbox.py

```diff
@@ -67,3 +71,76 @@ def mask2bbox(mask: np.ndarray) -> list[list[int]]:
         bboxes.append([x1, y1, x2, y2])
 
     return bboxes
+
+
+def convert_polygon_to_mask(shape: Polygon, width: int, height: int) -> np.ndarray:
+    """Convert polygon to mask.
+
+    Args:
+        shape (Polygon): Polygon to convert.
+        width (int): Width of image.
+        height (int): Height of image.
+
+    Returns:
+        np.ndarray: Generated mask from given polygon.
+    """
+    polygon = ShapeFactory.shape_as_polygon(shape)
+    contour = [[int(point.x * width), int(point.y * height)] for point in polygon.points]
+    gt_mask = np.zeros(shape=(height, width), dtype=np.uint8)
+    return cv2.drawContours(gt_mask, np.asarray([contour]), 0, 1, -1)
+
+
+def generate_bbox(
```

Comment:
(Just a comment, not change request)
Function name is misleading. It's kind of "purturb_bbox()", right? @sungchul2 
If this should be located in adapters/torch/modules/utils, it would be better to be more clear or general. It looks like Visual Prompt task specific logic.

Reply:
Yes, this function was to generate bboxes from given points with some perturbations when training.
Fitted box can be generated with `offset_bbox=0` as the default setting.
If it seems ambiguous, we can choose one of two: 
1. divide it into fitted version and perturbed version, and perturbed version can call fitted version
2. change function name like `generate_bbox_with_perturbation`

Reply:
Which is better? I think I'll go with your recommendation. :)

Reply:
I think 1 seems better, what do you think @goodsong81 ?

Reply:
BTW, do we need fitted version? It will be just one-liner.
like return [max(0, x1), max(0, y1), min(w, x2), min(h, y2)]

Reply:
This currently seems to always be used as 0, right?

Reply:
Anyway, Should it look like this?
```python
def generate_fitted_bbox(
    x1: int,
    y1: int,
    x2: int,
    y2: int,
    width: int,
    height: int,
) -> list[int]:
    """Generate bounding fitted box.

    Args:
        x1, y1, x2, y2 (int): Bounding box coordinates.
        width (int): Width of image.
        height (int): Height of image.

    Returns:
        list[int]: Generated bounding box.
    """
    return [
        max(0, x1),
        max(0, y1),
        min(width, x2),
        min(height, y2),
    ]


def generate_bbox_with_perturbation(
    x1: int,
    y1: int,
    x2: int,
    y2: int,
    width: int,
    height: int,
    offset_bbox: int = 0,
) -> list[int]:
    """Generate bounding box with perturbation.

    Args:
        x1, y1, x2, y2 (int): Bounding box coordinates.
        width (int): Width of image.
        height (int): Height of image.
        offset_bbox (int): Offset to apply to the bounding box, defaults to 0.

    Returns:
        list[int]: Generated bounding box.
    """

    def get_randomness(length: int) -> int:
        if offset_bbox == 0:
            return 0
        rng = np.random.default_rng()
        return int(rng.normal(0, min(int(length * 0.1), offset_bbox)))

    return generate_fitted_bbox(
        x1 + get_randomness(width),
        y1 + get_randomness(height),
        x2 + get_randomness(width),
        y2 + get_randomness(height),
        width,
        height,
    )
```

Reply:
Ah, fitted version is already there :)

Reply:
I did a simple function renaming (generate_bbox_with_perturbation) on this for now.